deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/cora/pvc_data.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  labels:
+    app: indexer-single
+    app.kubernetes.io/managed-by: kustomization
+    app.kubernetes.io/part-of: storetheindex
+  name: data
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Ti
```

Comment:
This volume is used for provider info only right? 

Would it make sense to look at its utilisation in other nodes and potentially reduce this? We can always grow it later but shrinking it is non trivial and would need data migration.

Reply:
The data stores do grow with recording all the ads ingested as well as keeping the mh counts for all contextIDs. ... and of course all that wonderful data-transfer FSM state.  On ago this is currently 24G... so would 100G be good?

Reply:
`100Gi` sounds good 👍